### ISSUE TITLE
DMN Editor - Marshaller support for 1.3 version

### ISSUE THREAD

**Issue by yesamer** [apache/incubator-kie-issues#3]
Currently, DMN Editor doesn't fully support the `1.3` DMN version, as shown in the below table:

<img width="1006" alt="Screenshot 2022-10-14 at 14 54 58" src="https://user-images.githubusercontent.com/16005046/198623117-7c6c7339-2077-4c87-bfb2-8746dc4d59d9.png">

Basically, DMN assets versions `1.1` and `1.3` are opened, managed and saved with DMN `1.2` specifications. 
More details [here](https://blog.kie.org/2020/11/dmn-editor-opens-1-1-and-1-3-assets-now.html).

Aim of this ticket is to enable full support for 1.3 DMN assets, modifying the DMN Editor marshaller to use `1.3` as the default target version, to open, manage and save DMN assets.

To summarize:
- The editor must be able to load and save `1.3` assets;
- The editor must automatically manage DMN `1.1` and DMN `1.2` assets to DMN `1.3` version;

**Comment by jomarko** [comment]
Are:
- the group element (kiegroup/kie-issues/issues/4)
- predefined function calls for comparing points in time and time intervals (kiegroup/kie-issues/issues/48)

The only additions to DMN 1.3?

Asking because of https://blog.viadee.de/en/this-is-new-in-dmn-1_3

**Comment by yesamer** [comment]
@jomarko From an editor perspective I think so, but feel free to correct me if I missing something :)

**Comment by jomarko** [comment]
@yesamer ok, then I reported this to track keywords and functions https://github.com/kiegroup/kie-issues/issues/48

**Comment by yesamer** [comment]
@tiagobento We can close this in favor of https://github.com/kiegroup/kie-issues/issues/220

We can use the collected data here to update https://github.com/kiegroup/kie-issues/issues/220 description
